src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs

```diff
@@ -365,6 +365,16 @@ public override void ExecuteCmdlet()
 
             if( !string.IsNullOrWhiteSpace(CertificatePath))
             {
+                if (!AzureSession.Instance.DataStore.FileExists(CertificatePath))
```

Comment:
Is that possible FileExists returns true even if CertificatePath is relative path?